ledger/src/blockstore_meta.rs

```diff
@@ -426,6 +470,60 @@ mod test {
         }
     }
 
+    #[test]
+    fn test_connected_flags_compatibility() {
```

Comment:
Not sure if this test if overkill or not

Reply:
actually, can we add a case for connected false equality with empty-flags? :sweat_smile:

Reply:
The existing tests look good.  Maybe also consider adding a test case for slot 0 and verifying its connected flag?